bin/rialto/runtime/src/millau_messages.rs

```diff
@@ -125,10 +128,16 @@ pub type ToMillauBlobExporter = HaulBlobExporter<
 pub struct ToMillauXcmBlobHauler;
 
 impl XcmBlobHauler for ToMillauXcmBlobHauler {
-	type MessageSender = pallet_bridge_messages::Pallet<Runtime, WithMillauMessagesInstance>;
-	type MessageSenderOrigin = RuntimeOrigin;
+	type Runtime = Runtime;
+	type MessagesInstance = WithMillauMessagesInstance;
 	type SenderAndLane = MullauSenderAndLane;
 
+	type ToSendingChainSender = crate::xcm_config::XcmRouter;
+	type CongestedMessage = DummyXcmMessage;
+	type UncongestedMessage = DummyXcmMessage;
+
+	type MessageSenderOrigin = RuntimeOrigin;
```

Comment:
@svyatonik 
sorry, unrelated, to this, 

but can we remove this `MessageSenderOrigin` we dont really use it for anything, 
I think now it just brings confusion to configuration, because it is hard-coded and not the real sender (e.g. sibling parachain)
```
pallet_xcm::Origin::from(MultiLocation::new(1, crate::xcm_config::UniversalLocation::get())).into()
...
pallet_xcm::Origin::from(MultiLocation::here()).into()
```

I think this `message_sender_origin` had its place when `send_message` was exposed as a extrinsic, but it is not now

Reply:
Yep, I agree there's a lot of things that may confuse in the configuration. I've been working on trying to make configuration easier in #1666. E.g. this origin is already gone. But I wanted to avoid pushing unnecessary changes to `polkadot-staging`, so it was "scheduled" for v2.

But this particular type is well isolated and if you think it'd be better to remove it in v1, I could prepare a separate PR.

Reply:
yes, I would even delete it directly in `polkadot-staging` now

Reply:
okay - will do soon :)